docs/resources/as_group.md

```diff
@@ -145,18 +123,19 @@ The following arguments are supported:
 * `desire_instance_number` - (Optional, Int) Specifies the expected number of instances. The default value is the
   minimum number of instances. The value ranges from the minimum number of instances to the maximum number of instances.
 
-* `min_instance_number` - (Optional, Int) Specifies the minimum number of instances. The default value is 0.
+* `min_instance_number` - (Optional, Int) Specifies the minimum number of instances. Defaults to **0**.
```

Comment:
Defaults to `0`.
number use (``).

Reply:
Please also check the relevant formats in other places

Reply:
keep status quo